examples/chart/teleport-relay/values.yaml

```diff
@@ -0,0 +1,606 @@
+  deployment: {}
+  # annotations.pod(object) -- are annotations to set on the Pods.
+  pod: {}
+  # annotations.podDisruptionBudget(object) -- are annotations to set on the
+  # podDisruptionBudget.
+  podDisruptionBudget: {}
+  # annotations.secret(object) -- are annotations to set on the Secret.
+  secret: {}
+  # annotations.serviceAccount(object) -- are annotations to set on the
+  # ServiceAccount.
+  serviceAccount: {}
+
+# image(string) -- sets the container image used for Teleport Community Edition
+# Agent pods created by the chart.
+#
+# You can override this to use your own Teleport image rather than a
+# Teleport-published image.
+#
+# By default, the image contains only the Teleport application and its runtime
+# dependencies, and does not contain a shell.
+#
+# This setting only takes effect when [`enterprise`](#enterprise) is `false`.
+# When running an enterprise version, you must use
+# [`enterpriseImage`](#enterpriseImage) instead.
+image: public.ecr.aws/gravitational/teleport-distroless
+
+# enterpriseImage(string) -- sets the container image used for Teleport Enterprise
+# agent pods created by the chart.
+#
+# You can override this to use your own Teleport image rather than a
+# Teleport-published image.
+#
+# By default, the image contains only the Teleport application and its runtime
+# dependencies, and does not contain a shell.
+#
+# This setting only takes effect when [`enterprise`](#enterprise) is `true`.
+# When running an enterprise version, you must use [`image`](#image) instead.
+enterpriseImage: public.ecr.aws/gravitational/teleport-ent-distroless
+
+# imagePullSecrets(list) -- is a list of secrets containing authorization tokens
+# which can be optionally used to access a private Docker registry.
+#
+# See the [Kubernetes
+# reference](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)
+# for more details.
+imagePullSecrets: []
+
+# imagePullPolicy(string) -- sets the pull policy for any pods created by the
+# chart. See [the Kubernetes
+# documentation](https://kubernetes.io/docs/concepts/containers/images/#updating-images)
+# for more details.
+imagePullPolicy: IfNotPresent
+
+# topologySpreadConstraints(list) -- sets the topology spread constraints for
+# any pods created by the chart. See [the Kubernetes
+# documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+# for more details.
+#
+# When unset, the chart defaults to a soft topology spread constraint that tries
+# to spread pods across hosts and zones.
+#
+# ```
+# topologySpreadConstraints:
+#   - maxSkew: 1
+#     topologyKey: kubernetes.io/hostname
+#     whenUnsatisfiable: ScheduleAnyway
+#     labelSelector:
+#       matchLabels: # dynamically computed
+#   - maxSkew: 1
+#     topologyKey: topology.kubernetes.io/zone
+#     whenUnsatisfiable: ScheduleAnyway
+#     labelSelector:
+#       matchLabels: # dynamically computed
+# ```
+topologySpreadConstraints: []
```

Comment:
Can you set the TSC by default to the zone/node one in the example? With scheduleNayway this won't block anything but make sure we do best effort availability by default.